src/Visualization/Python/GenerateXdmf.py

```diff
@@ -243,10 +243,12 @@ def parse_args():
         formatter_class=ap.ArgumentDefaultsHelpFormatter)
     parser.add_argument(
         '--file-prefix',
+        '-i',
         required=True,
         help="The common prefix of the H5 volume files to load")
     parser.add_argument(
-        '--output',
+        '--output-filename',
```

Comment:
This is inconsistent with basically every command line utility I know of. Please leave it as `--output`. I'm fine with `-o` as short for output since that's standard